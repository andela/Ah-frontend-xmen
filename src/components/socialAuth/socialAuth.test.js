import React from 'react';
import { shallow } from 'enzyme';
import SocialAuth from './socialAuth';


describe('<SocialAuth', () => {
  it('should render without crushing', () => {
    const wrapper = shallow(<SocialAuth />);
    expect(wrapper).toMatchSnapshot();
  });
});
