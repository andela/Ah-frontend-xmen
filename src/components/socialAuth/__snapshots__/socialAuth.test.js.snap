// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SocialAuth should render without crushing 1`] = `ShallowWrapper {}`;
